.github/workflows/ubuntu_build.yml

```
@@ -14,26 +14,30 @@ on:
 
 permissions:
   contents: read
-env: 
-    DPDK_VERSION: 25.03
+
 jobs:
   changes:
     runs-on: ubuntu-latest
     permissions:
       pull-requests: read
     outputs:
-      changed: ${{ steps.filter.outputs.ubuntu_build == 'true' }}
+      base: ${{ steps.filter.outputs.ubuntu_build == 'true' }}
+      afxdp_build: ${{ steps.filter.outputs.afxdp_build == 'true' }}
     steps:
```

Move env from end to this place for easy to understand flow:

```suggestion
    env:
      SETUP_ENVIRONMENT: 0
      BUILD_AND_INSTALL_EBPF_XDP: 0
      BUILD_AND_INSTALL_DPDK: 0
      BUILD_ICE_DRIVER: 0 # Only for CICD action builds
      BUILD_AND_INSTALL_ICE_DRIVER: 0
    steps:    
```